database.go

```diff
@@ -0,0 +1,78 @@
+package authenticatedblockstore
+
+import (
+	"fmt"
+	"log"
+)
+
+type DataBase struct {
+	tableManager *TableManager
+	logManager   *LogManager
+}
+
+// Exported Package APIs.
+// These are called by: package.api()
+//
+//
+func Create() (*DataBase, error) {
+	fmt.Println("Creating a new DataBase, returning a handle to the caller.")
+	var newDB *DataBase
+
+	tb, err := NewTableManager()
+	if err != nil {
+		log.Fatal(err)
+	}
+	newDB.tableManager = tb
```

Comment:
Can you just write this as:

newDB.tb, err := NewTableManager()
...

and avoid the temporary?

Reply:
Right, made the change.